sw/device/lib/testing/flash_ctrl_testutils.h

```diff
@@ -121,4 +121,21 @@ bool flash_ctrl_testutils_read_page(
     dif_flash_ctrl_partition_type_t partition_type, uint32_t word_count,
     uint32_t delay);
 
+/**
+ * Sets the flash default configuration.
+ *
+ * @param flash_state A flash_ctrl state handle.
+ * @param rd_en Default read enable.
+ * @param prog_en Default program enable.
+ * @param erase_en Default page erase enable.
+ * @param scramble_en Default scramble enable.
+ * @param ecc_en Default ECC enable.
+ * @param high_endurance_en Default high endurance enable
+ */
+void flash_ctrl_utils_default_region_access(dif_flash_ctrl_state_t *flash_state,
```

Comment:
nit: @timothytrippel pointed out that the prefix should be `flash_ctrl_testutils_`. From https://docs.opentitan.org/sw/device/lib/testing/

Reply:
sounds good. btw i accidentally ran clang-format on a build file..hence all the failures right now..argh.